Fix broken path in README.md (#81)
cortinico/kotlin-android-template: README.md

This template is using [**Gradle Kotlin DSL**](https://docs.gradle.org/current/userguide/kotlin_dsl.html) as well as the [Plugin DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) to setup the build.

> Dependencies are centralized inside the [libs.versions.toml](gradle/libs.versions.toml) file in the `gradle` folder. This provides convenient auto-completion when writing your gradle files.

Reviewer: ```suggestion
Dependencies are centralized inside the Gradle Version Catalog in the [libs.versions.toml](gradle/libs.versions.toml) file in the `gradle` folder.
```
